internal/utils/multiInput.go

```diff
@@ -0,0 +1,245 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/charmbracelet/lipgloss"
+	"os"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/cursor"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+var (
+	titleStyle       = TitleStyle.Copy().MarginLeft(2)
+	descriptionStyle = BlurredStyle.Copy().MarginLeft(2).Foreground(Colors.BrightGrey)
+
+	inputBoxStyle = DocStyle.Copy().
+			PaddingLeft(2).
+			Border(lipgloss.NormalBorder(), false, false, false, true).
+			BorderForeground(Colors.DimYellow)
+
+	focusedPromptStyle = FocusedStyle.Copy().Bold(true)
+	blurredPromptStyle = focusedPromptStyle.Copy().Foreground(Colors.White)
+	placeholderStyle   = BlurredStyle.Copy()
+)
+
+type MultiInput struct {
+	title       string
+	description string
+	inputs      []InputField
+
+	inputModels    []textinput.Model
+	inputsRequired bool
+
+	cursorMode cursor.Mode
+	focusIndex int
+	done       bool
+}
+
+type InputField struct {
+	Name        string
+	Placeholder string
+	Value       string
+}
+
+func NewMultiInput(title, description string, required bool, inputs ...InputField) MultiInput {
+	m := MultiInput{
```

Comment:
I don't think this interface is perfect by any means, but i dont think it makes sense to try to make the perfect abstraction right now given there's only one place we're using it

Reply:
Ya I think this is totally fine right now. After we get more of the CLI changes down we can decide if we want to shift more into this custom tea model direction or if we want to try to adapt huh for what we want here.

I don't think it's the end of the world to be in a place where we use a bit of both too since they are just charm packages that do connect to each other. A form is just a huh model